src/tests/inbound/createInbound/exportItems.test.ts

```diff
@@ -127,5 +130,107 @@ test.describe('Export all incomming items', () => {
+    });
+
+    await test.step('Select item to receive', async () => {
+      await receivingPage.receivingStep.isLoaded();
+      await receivingPage.receivingStep.table
+        .row(1)
+        .receivingNowField.textbox.fill('12');
+    });
+
+    await test.step('Go to Check page', async () => {
+      await receivingPage.nextButton.click();
+    });
+
+    await test.step('Receive shipment', async () => {
+      await receivingPage.checkStep.isLoaded();
+      await receivingPage.checkStep.receiveShipmentButton.click();
+      await stockMovementShowPage.isLoaded();
+    });
+
+    await test.step('Go to inbound list page', async () => {
+      await inboundListPage.goToPage();
+    });
+
+    await test.step('Download file', async () => {
+      const { fullFilePath } = await inboundListPage.downloadAllIncomingItems();
+      filePath = fullFilePath;
+    });
+
+    await test.step('Read file', async () => {
+      downloadedTemplateFile = WorkbookUtils.read(filePath);
+      workbooks.push(downloadedTemplateFile);
+    });
+
+    await test.step('Assert 1 item from stock movement is included in the epxort file', async () => {
+      const { data } = await stockMovementService.getStockMovement(INBOUND_ID);
+      const fileData = downloadedTemplateFile.getData();
+
+      const stockMovementItemsFromFile = fileData.filter(
+        (it) => it[5] === data.identifier
+      );
+      expect(stockMovementItemsFromFile).toHaveLength(1);
+    });
+
+    await test.step('Go to inbound view page', async () => {
+      await stockMovementShowPage.goToPage(INBOUND_ID);
+      await stockMovementShowPage.isLoaded();
+    });
+
+    await test.step('Go to shipment receiving page', async () => {
+      await stockMovementShowPage.receiveButton.click();
+      await receivingPage.receivingStep.isLoaded();
+    });
+
+    await test.step('Select item to receive', async () => {
+      await receivingPage.receivingStep.isLoaded();
+      await receivingPage.receivingStep.table
+        .row(2)
+        .receivingNowField.textbox.fill('12');
+    });
+
+    await test.step('Go to Check page', async () => {
+      await receivingPage.nextButton.click();
+    });
+
+    await test.step('Receive shipment', async () => {
+      await receivingPage.checkStep.isLoaded();
+      await receivingPage.checkStep.receiveShipmentButton.click();
+      await stockMovementShowPage.isLoaded();
+    });
+
+    await test.step('Go to inbound list page', async () => {
+      await inboundListPage.goToPage();
+    });
+
+    await test.step('Download file', async () => {
+      const { fullFilePath } = await inboundListPage.downloadAllIncomingItems();
+      filePath = fullFilePath;
+    });
+
+    await test.step('Read file', async () => {
+      downloadedTemplateFile = WorkbookUtils.read(filePath);
+      workbooks.push(downloadedTemplateFile);
+    });
+
+    await test.step('Assert received items from stock movement are not included in the epxort file', async () => {
```

Comment:
It can be split into 3 tests for asserting shipped, partially received and received inbound but as the original test was named 'Export all incomming items should include items shipped not received' I included all cases in 1 test.